src/components/Apps/AppsContent.jsx

```diff
@@ -29,14 +36,19 @@ class AppsContent extends Component {
     return (
       <div className="coz-nav-pop-content">
         <ul className="coz-nav-group">
-          {isMobile && homeApp && <AppItem app={homeApp} useHomeIcon />}
+          {isMobile &&
+            homeApp && (
+              <AppItem app={homeApp} useHomeIcon onAppSwitch={onAppSwitch} />
```

Comment:
Did you consider going through a beforeUnload event instead of adding a new props ?

Reply:
No I didn't, but if I use this event listener I would not be able to disable this closing behaviour by navigating from web app to a web app since this info is checked in the `AppItem` component